Nodejs/Tests/AnalysisTests/AnalysisFile.cs

```diff
@@ -131,6 +131,20 @@ static class Analysis {
             List<AnalysisFile> files = new List<AnalysisFile>();
             foreach (var file in Directory.GetFiles(directory, "*", SearchOption.AllDirectories)) {
```

Comment:
Really the tool exists to do comparisons between different versions of the analyzer.  It certainly doesn't run anywhere in an automated way because you really need to get your baseline, make the changes, and then select what you want to compare it to.

For keeping the product code and test code in sync here what I'd do is move the file check into a common helper where both NTVS and AnalysisDriver can both call into it (e.g. JsAnalyzer).  Then I'd have them both manage the nested modules limit independently - with AnalysisDriver updated to take it from the command line, and NTVS taking it from user settings somewhere.